config/locales/en.yml

```diff
@@ -129,6 +129,14 @@ en:
     explore: Explore
     log_in: Log in with Wikipedia
     sign_up: Sign up with Wikipedia
+    registration_guidance:
+      username_selection_advice:
+        heading: Selecting a Username
+        avoid_offensive_usernames: Avoid usernames that could be considered offensive, misleading, disruptive, or promotional.
```

Comment:
"Do no choose a username that is offensive, misleading, disruptive, or promotional."